Add orbital order to *sqr.dat output description (#747)
dftbplus/dftbplus: doc/dftb+/manual/dftbp.tex

  \end{itemize}
\end{itemize}

> Each column / row of a quadratic dense matrix belongs to an (atom, orbital)

Reviewer: What's a quadratic matrix (do you mean square)?

Reviewer: ```suggestion
Each column / row of a square dense matrix belongs to an (atom, orbital)
```

index pair, with the orbital index running faster. The order of the atoms
follows their order in the input geometry. The orbitals are ordered by

> increasing ($l$, $m$) angular quantum numbers of the real tesserals with $m$

Reviewer: Not necessarily, as the l ordering is depending on shell ordering (in the case of extended basis specification).

Reviewer: ```suggestion
increasing magnetic quantum number ($m$) of the angular shell ($l$), using real tesserals functions. The angular shells
are ordered according to the input to the calculation. In most cases this leads to orbitals ordered by increasing ($l$, $m$), but for non-minimal or reordered basis functions the shell order ($l$) may not necessarily be in increasing order.
```

> running faster. For a given atom it results in the orbital order

Reviewer: ```suggestion
For the usual minimal basis, the standard orbital order runs as
```

\begin{equation*}
  s,\, p_y,\, p_z,\, p_x,\, d_{xy},\, d_{yz},\, d_{z^2},\, d_{xz},\,
  d_{x^2-y^2},\, f_{y(3x^2-y^2)},\, f_{xyz},\, f_{yz^2},\, f_{z^3},\, f_{xz^2},
  f_{z(x^2-y^2)},\, f_{x(x^2-3y^2)} \text,
\end{equation*}

> where only the orbitals present in the basis of the atom are considered.

Reviewer: ```suggestion
for an atom, where only the orbitals present in the selected basis of the atom are considered.
```

  \end{itemize}
\end{itemize}

Each column / row of a square dense matrix belongs to an (atom, orbital)
index pair, with the orbital index running faster. The order of the atoms
follows their order in the input geometry. The orbitals are ordered by
increasing magnetic quantum number ($m$) of the angular shell ($l$), using real tesserals functions. The angular shells
are ordered according to the input to the calculation. In most cases this leads to orbitals ordered by increasing ($l$, $m$), but for non-minimal or reordered basis functions the shell order ($l$) may not necessarily be in increasing order.
running faster. For a given atom it results in the orbital order
\begin{equation*}
  s,\, p_y,\, p_z,\, p_x,\, d_{xy},\, d_{yz},\, d_{z^2},\, d_{xz},\,
  d_{x^2-y^2},\, f_{y(3x^2-y^2)},\, f_{xyz},\, f_{yz^2},\, f_{z^3},\, f_{xz^2},
  f_{z(x^2-y^2)},\, f_{x(x^2-3y^2)} \text,
\end{equation*}

> for an atom, where only the orbitals present in the selected basis of the atom are considered. In other more general cases, while $m$ is always in increasing order, $l$ may be reordered, have multiple shells, or be lacking particular $l$ values.

Reviewer: ```suggestion
for an atom, where only the orbitals present in the selected basis of the atom are considered. In other more general cases, while $m$ is always in increasing order for a given shell, $l$ may be reordered, have multiple shells, or be lacking particular $l$ values.
```
